app/views/admin/applications/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<br><h2>Admin View for Application <%= @application.id %></h2>
+<p> <b>Applicant Name: </b><%= @application.name  %></p> 
+<p> <b>Address: </b> <%= @application.address_join %></p>
+<p> <b>Description of Qualification: </b><%= @application.description %></p> 
+<p> <b>Application Status: </b><%= @application.status %></p> 
+<br>
+
+<h3>All Pets Applied For:</h3>
+<% @application.application_pets.each do |app_p| %>
+    <h3><%= app_p.pet.name %></h3>
+    <h4><%= "** Approved!**" if app_p.approved(@application.id) %></h4>
+    <h4><%= "Pet has been approved on a different application!" if app_p.pet.application_approved? && !app_p.approved(@application.id) %></h4>
+    <h4><%= "** Rejected **" if app_p.rejected(@application.id) %></h4>
+    <p> <%= "Status: Adoptable" if app_p.pet.adoptable%> <%= "Status: Not Adoptable" if !app_p.pet.adoptable && !app_p.approved(@application.id)%></p> 
+    <%= button_to "Approve #{app_p.pet.name}", {controller: "application_pets", action: "update", application_id: @application.id, pet_id: app_p.pet.id, approve: true}, method: :patch if !app_p.pet.application_approved? && !app_p.approved(@application.id) && !app_p.rejected(@application.id)%>
+    <%= button_to "Reject #{app_p.pet.name}", {controller: "application_pets", action: "update", application_id: @application.id, pet_id: app_p.pet.id, approve: false}, method: :patch if !app_p.pet.application_approved? && !app_p.approved(@application.id) && !app_p.rejected(@application.id)%>
```

Comment:
I like how you've organized this rendering on this view, but I wonder if you could get these if statements down to 2 conditions rather than 3, for the sake of developer empathy. Maybe a model method in the application_pets model that checks both whether an app_pet is approved OR rejected?